flake.nix

```diff
@@ -113,7 +113,10 @@
                               else
                                 [ ]
                           )
-                          [ ])
+                          [{
+                            name = attrName;
+                            value = throw "failed";
```

Comment:
I wonder if there's a way to reuse the error that would have occurred if we recursed here, so that it's not just a context-less "failed"?

Reply:
Not easily. We could do something like `value = ... whatever value caused the error ...` but that's a bit error-prone. Maybe something like
```
value = assert attrs.type != null; throw "failed";
```